Makefile

```diff
@@ -0,0 +1,10 @@
+ROOT_PATH        ?= "$(shell pwd)"
+ORG              ?= ansibleplaybookbundle
+TAG              ?= latest
+REGISTRY         ?= docker.io
+
+kubevirt-apb:
+	$(eval IMAGE_NAME?=kubevirt-apb)
+	pushd $(ROOT_PATH); docker build --tag $(REGISTRY)/$(ORG)/$(IMAGE_NAME):$(TAG) -f ${ROOT_PATH}/kubevirt-apb/Dockerfile $(ROOT_PATH); popd
```

Comment:
Don't we want to use the `apb tool` for this action?

Reply:
We do, but there's an issue with the apb tool expecting apb.yml to be in ```./apb.yml``` when this needs to be built from outside the kubevirt-apb directory.  This will work around the issue.